reference/extensions/custom_commands.rst

```diff
@@ -63,27 +61,31 @@ Main decorator to declare a function as a new Conan command. Where the parameter
     import json
 
     from conan.api.conan_api import ConanAPI
-    from conan.api.output import ConanOutput
+    from conan.api.output import cli_out_write, ConanOutput
     from conan.cli.command import conan_command
 
+
     def output_json(msg):
-        return json.dumps({"greet": msg})
+        cli_out_write(cli_json.dumps({"greet": msg}))
+
+    def output_text(msg):
+        ConanOutput().info(msg)
```

Comment:
output text shouldn't use ``ConanOutput()``, that goes to sys.stderr, it should probably use ``cli_out_write()`` too

Reply:
but `ConanOutput()` is kind of "non-parseable" user output right? Why should it use `cli_out_write()`? This is what was confusing to me and what I'd like to clarify with this PR

Reply:
If it is a "formatter", it is intended for some kind of "final message" result. Even if in the text format is not machine readable, it should go to ``stdout``, not to ``stderr``, because it is the output of the command.

If that is not the case, and it is just informational message, it can be put in the command itself, and not in the formatter.

Reply:
updated!

Reply:
I still see the ``ConanOutput().info()`` and not ``cli_out_write()``, did you forget to push changes?